src/main/java/ch/njol/skript/expressions/ExprSpawner.java

```diff
@@ -0,0 +1,131 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+
+package ch.njol.skript.expressions;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.bukkit.Material;
+import org.bukkit.block.Block;
+import org.bukkit.block.CreatureSpawner;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.classes.Converter;
+import ch.njol.skript.aliases.Aliases;
+import ch.njol.skript.classes.Changer;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.entity.EntityData;
+import ch.njol.skript.entity.EntityType;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+
+@Name("Spawner Type")
+@Description("Retrieves, sets, or resets the spawner's entity type")
+@Examples({"broadcast \"%spawner's entity type%\""})
```

Comment:
```suggestion
@Examples("set entity type of targeted block to entitytype-argument")
```
remove { } because the string is single, also it was a wrong code

Reply:
entitytype-argument feels just wrong, I'd rather see:
```vb
broadcast "%target block's creature type%"
```

Reply:
Well then just add a full and simple command, also use `arg-1`. I'd rather an useful example.

Reply:
Added a new example
```
on right click:
	if event-block is spawner:
		send "Spawner's type is %target block's creature type%"
```

Could not do ``on right click of spawner`` or ``on right click of spawner block`` for some odd reason